src/utils/helper-funcs.jsx

```diff
@@ -0,0 +1,24 @@
+export const sortFunc = (articlesArr, order) => {
+  function compare(a, b) {
+    const voteA = a.votes;
+    const voteB = b.votes;
+    let comparison = 0;
+    if (!order || order === "ASC") {
+      if (voteA > voteB) {
+        comparison = -1;
+      } else if (voteA < voteB) {
+        comparison = 1;
+      }
+    } else if (order === "DESC") {
+      if (voteA > voteB) {
+        comparison = 1;
+      } else if (voteA < voteB) {
+        comparison = -1;
+      }
+    }
+    return comparison;
+  }
+  return articlesArr.sort(compare);
+};
+
+export default sortFunc;
```

Comment:
Do you have tests for this helper function? 👀  a great chance to flex those testing skills!